src/diffusers/utils/import_utils.py

```diff
@@ -21,6 +21,7 @@
 from collections import OrderedDict
 from typing import Union
 
+from huggingface_hub.utils import is_jinja_available  # noqa: F401
```

Comment:
Just to be sure this is supported since huggingface_hub `0.10` ? That's currently our minimum required version for the Hub.

Reply:
Ah yeah you answered that above

Reply:
[Yes it is !](https://github.com/huggingface/huggingface_hub/blob/v0.10.0/src/huggingface_hub/utils/__init__.py#L56)